### PR TITLE
Add mail tester

### PR DESCRIPTION
**Pull request by ayan4m1**
This PR adds a "Test" button to the Admin > Settings > Mail tab. It tries to send a generic "testing" email to the currently logged in user's email address. References #1233.

### CODE REVIEW

File: app/Http/Controllers/Admin/Settings/MailController.php [hunk 1]
```diff
@@ -111,4 +115,26 @@ public function update(MailSettingsFormRequest $request): RedirectResponse
...
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function test(Request $request): RedirectResponse
+    {
+        try {
+            Notification::route('mail', $request->user()->email)
+                ->notify(new MailTested($request->user()));
+        } catch (Exception $exception) {
+            $this->alert->danger(trans('base.mail.test_failed'))->flash();
```
Comment (DaneEveritt) [inline app/Http/Controllers/Admin/Settings/MailController.php]: Think it makes sense to catch some type of mailer exception and display the message here? Might help us save a step when the user comes asking saying 'There was an error'.

File: resources/themes/pterodactyl/admin/settings/mail.blade.php [hunk 2]
```diff
@@ -98,7 +98,10 @@
                         </div>
                         <div class="box-footer">
                             {{ csrf_field() }}
-                            <button type="submit" name="_method" value="PATCH" class="btn btn-sm btn-primary pull-right">Save</button>
+                            <div class="pull-right">
+                                <a href="{{ route('admin.settings.mail.test') }}" class="btn btn-sm btn-success">Test</a>
```
Comment (DaneEveritt) [inline resources/themes/pterodactyl/admin/settings/mail.blade.php]: UX seems confusing, if they change a value and click 'Test' but not 'Save' it'll try sending using the old information?

**Comment by DaneEveritt** [comment]
@ayan4m1 please resolve the conflicts 👍

**Comment by ayan4m1** [comment]
Done.

**Pull request merged**